spanner/value.go

```diff
@@ -877,6 +877,96 @@ func (n *PGNumeric) UnmarshalJSON(payload []byte) error {
 	return nil
 }
 
+// NullProto represents a Cloud Spanner PROTO that may be NULL.
+type NullProto struct {
+	ProtoVal proto.Message // ProtoVal contains the value when it is non-NULL, and nil when NULL.
```

Comment:
When NULL: Is it really `nil`? Or is the zero value of `proto.Message`? (is `proto.Message` an interface or a struct?)

Reply:
`proto.Message` is an interface. The zero value or default value of an interface is `nil`.

Reply:
Updated the comment to "when Valid is true"